src/mol-model/structure/query/queries/atom-set.ts

```diff
@@ -0,0 +1,44 @@
+/**
```

Comment:
Not sure if we really need this ported from molql or if there is something like it already in molstar. @dsehnal will know.

Reply:
ping @dsehnal